docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -1,8 +1,81 @@
 ---
-title: "Kubernetes agents"
+title: "Running Dagster+ agents on Kubernetes"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 30
 sidebar_label: "Kubernetes"
 ---
 
-# Kubernetes agents
\ No newline at end of file
+# Running Dagster+ agents on Kubernetes
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) on a [Kubernetes](https://kubernetes.io) cluster.
+
+## Installation
+
+
+### Prerequisites
+
+You'll need [a Kubernetes cluster. This can be a self-hosted Kubernetes cluster or a managed offering like [Amazon EKS](https://aws.amazon.com/eks/), [Azure AKS](https://azure.microsoft.com/en-us/products/kubernetes-service), or [Google GKE](https://cloud.google.com/kubernetes-engine).
+
+We recommend installing the Dagster+ agent using [Helm](https://helm.sh).
+
+## Step 1: Create a Kubernetes namespace
+
+```shell
+kubectl create namespace dagster-cloud
+```
+
+## Step 2: Create an agent token secret
+
+[Generate an agent token](dagster-plus/deployment/tokens) and set it as a Kubernetes secret:
+
+```shell
+kubectl --namespace dagster-cloud create secret generic dagster-cloud-agent-token --from-literal=DAGSTER_CLOUD_AGENT_TOKEN=<token>
+```
+
+## Step 3: Add the Dagster+ agent Helm chart repository
+
+```shell
+helm repo add dagster-cloud https://dagster-io.github.io/helm-user-cloud
+helm repo update
+```
+
+## Step 4: Install the Dagster+ agent Helm chart
+
```

Comment:
The OG guide included a step for a `values.yaml` - is that still needed?

Reply:
+1 to having a values.yaml example included here

Reply:
https://github.com/dagster-io/dagster/pull/24030/commits/194a07d371768ca9c8d129928bcabecc6de8e733 switches from `--set` to `--values` - which do we prefer?

Reply:
I slightly prefer values.yaml, in part because of mistakes some folks have made with --set, but it's not a huge preference